### ISSUE TITLE
Callback not working  - sort-lines:sort

### ISSUE THREAD

**Issue by Mayccoll** [cakecatz/flex-toolbar#17]
Hi, great tool, tnks

I'm having an issue, sort lines is not working at all.
 ```json
{
    "type": "button",
    "icon": "sort-alpha-asc",
    "callback": "sort-lines:sort",
    "tooltip": "Sort Lines",
    "iconset": "fa"
  },
```
![out](https://cloud.githubusercontent.com/assets/2009698/7635438/9ec4144a-fa26-11e4-820e-818cd8e925d1.gif)

**Comment by jerone** [comment]
This is probably related to https://github.com/suda/tool-bar/pull/36

**Comment by cakecatz** [comment]
Fixed by suda/tool-bar#36
